spec/features/web_intake/new_single_filer_spec.rb

```diff
@@ -204,19 +198,7 @@
 
     # Itemizing
     expect(page).to have_selector("h1", text: "Would you like to itemize your deductions for #{current_tax_year}?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In #{current_tax_year}, did you pay any medical, dental, or prescription expenses?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In #{current_tax_year}, did you make any charitable contributions?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In #{current_tax_year}, did you have any income from gambling winnings, including the lottery?")
     click_on "No"
-    expect(page).to have_selector("h1", text: "In #{current_tax_year}, did you pay for any eligible school supplies as a teacher, teacher's aide, or other educator?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In #{current_tax_year}, did you pay any state, local, real estate, sales, or other taxes?")
-    click_on "Yes"
-    expect(page).to have_selector("h1", text: "In #{current_tax_year}, did you receive a state or local income tax refund?")
-    click_on "Yes"
```

Comment:
as far as we could tell, there wasn't any test coverage of the gating of itemizing-related questions, so we made the single filer spec cover the non-itemizing case